ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -494,100 +495,106 @@ export default function ConfirmationPage({
 
   return (
     <ConfirmContextProvider>
-      <div className="confirmation-page">
-        <Header
-          confirmation={pendingConfirmation}
-          isSnapCustomUIDialog={isSnapCustomUIDialog}
-          onCancel={handleSnapDialogCancel}
-        />
-        <Box
-          className="confirmation-page__content"
-          padding={isSnapCustomUIDialog ? 0 : 4}
-          style={{
-            overflowY: 'auto',
-          }}
-        >
-          {isSnapCustomUIDialog ? (
-            <SnapUIRenderer
-              snapId={pendingConfirmation?.origin}
-              interfaceId={pendingConfirmation?.requestData.id}
-              isPrompt={isSnapPrompt}
-              inputValue={
-                isSnapPrompt && inputStates[pendingConfirmation?.type]
+      <AlertContextProvider
+        pendingConfirmation={pendingConfirmation}
+        onSubmit={!templatedValues.hideSubmitButton && handleSubmit}
+        onCancel={templatedValues.onCancel}
+      >
```

Comment:
Only change in this file is adding `< AlertContextProvider`

Reply:
I turned on `Hide whitespace` for this one 😄